cmd/stack_lint.go

```diff
@@ -29,7 +29,10 @@ func newStackLintCmd(rootConfig *RootCommandConfig) *cobra.Command {
 		Long: `Check that the structure of your stack is valid. Error messages indicate critical issues in your stack structure, such as missing files, directories, or stack variables. Warning messages suggest optional stack enhancements.
 
 		Run this command from the base directory of your stack, or specify the path to your stack.`,
-		Example: `  appsody stack lint [path/to/my-stack]
+		Example: `  appsody stack lint
+		Checks the structure of the stack in the current directory"
+		
+		appsody stack lint [path/to/my-stack]
```

Comment:
Shouldn't path/to/my-stack not have square brackets around it, as this is in an example command?

Reply:
will update